setup.py

```diff
@@ -8,7 +8,7 @@
 
 _base_extras = [
     'django>=4',
-    'django-allauth',
+    'django-allauth>=0.56.1',
```

Comment:
```suggestion
    # Required for "allauth.account.middleware.AccountMiddleware"
    'django-allauth>=0.56.0',
```

It is a good idea to justify the version bound here though.